### DISCUSSION TITLE
arkadaşlar bu github görevleri nasıl yapılıyor?

### DISCUSSION THREAD

**Discussion by tokmakci53** [IBAX-io/go-ibax#1032] [Soru Cevap - Türkçe] [answered]
yardımm

**Comment by mstfcnky** [comment]
Bağlantıları retwet yapmalısın

**Comment by mstfcnky** [comment]
Başka yolu var mı bende bilmiyorum

**Comment by Yusufknk** [comment, marked as answer]
Githubta star ve work yapmalısınız. Nasıl yapılacağı ile ilgili İbax Türkiye telegram grubunda sabitlenmiş mesajlarda video mevcut.

**Reply by ZiyaGungor** [reply]
Video da olan uygulamanın aynısını yaptık ama puan yansımadı hesaba

**Comment by hes91** [comment]
Airdrop sayfasında github sekmesinde star ve fork yaparak puanınızı alanilirsiniz.

**Comment by Msyn35** [comment]
ibax türkiye telegram kanalında sabitlenen mesajların içerisinde github la ilgili video mevcut.

**Reply by Eddyhunter07** [reply]
Evet
